docs: add windows related content (#8903)
langflow-ai/langflow: docs/docs/Contributing/contributing-how-to-contribute.md

* [Node.js](https://nodejs.org/en/download/package-manager)

> * [Make](https://www.gnu.org/software/make/#documentation)
> 
> :::important
> Make is not required for Windows installations. Installing and running Windows from source uses node and uv.
> Scripts are also provided in the `/scripts/windows` directory of the Langflow project.
> :::
> 

Reviewer: ```suggestion
* [Make](https://www.gnu.org/software/make/#documentation)

:::tip Windows
For Windows installations, you don't need need Make, and you can find [Windows scripts](https://github.com/langflow-ai/langflow/tree/main/scripts/windows) in the Langflow repository.
:::
```

Important is a little strong here.
Another alternative would be to use tabs (One tab for Linux and macOS, one tab for Windows) with the prereqs/info for each OS on each tab.

### Run Langflow from source

If you're not developing, but want to run Langflow from source after cloning the repo, run the following commands.

<Tabs groupId="os">

>   <TabItem value="macOS/Linux" label="macOS/Linux" default>

Reviewer: ```suggestion
  <TabItem value="Linux and macOS" label="Linux and macOS" default>
```

This label avoids starting with a lowercase letter and it is more readable.

1. To run Langflow from source, run the following command:
    ```bash
    make run_cli
    ```

>     This command:

Reviewer: ```suggestion
    This command does the following:
```

    - Installs frontend and backend dependencies
    - Builds the frontend static files
    - Starts the application with default settings

    The Langflow frontend is available at `http://localhost:7860/`.

  </TabItem>
  <TabItem value="Windows" label="Windows">

> To run Langflow from source on Windows, you can use the Langflow project's included scripts, or run the commands in the terminal.

Reviewer: I think these 2 options should be another set of nested tabs.

1. To run Langflow with the included scripts, navigate to the `scripts/windows` directory.
Two scripts are available to install and start Langflow.

2. Run Langflow with one of the scripts.

    * To install and start Langflow with a Windows Batch file, double-click `build_and_run.bat`.
    * To install and start Langflow with the Powershell script, run:

        ```ps
        .\build_and_run.ps1
        ```

**Alternatively**, to run Langflow from source with the Windows Command Line or Powershell, do the following.

1. Run the following commands to build the Langflow frontend.
    ```
    cd src/frontend
    npm install
    npm run build
    npm run start
    ```

2. Copy the contents of the built `src/frontend/build` directory to `src/backend/base/langflow/frontend`.

3. To start Langflow, run the following command.
    ```
    uv run langflow run
    ```

    The frontend is served at http://localhost:7860.

  </TabItem>
</Tabs>

### Set up your Langflow development environment

<Tabs groupId="os">
  <TabItem value="macOS/Linux" label="macOS/Linux" default>

1. To set up the Langflow development environment, run the following command:
    ```bash
    make init
    ```

>     This command sets up the development environment by:

Reviewer: ```suggestion
    This command sets up the development environment by doing the following:
```
